_docs/developer/testing/cypress_cloud.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Cypress Cloud
+---
+Submitty has an open source Cypress license to record and run tests via Cypress Cloud. Cypress records tests to [Cypress Cloud](https://cloud.cypress.io/projects/es51qa), where you can view a video of the tests running, and if they failed, a screenshot of the website at the point of failure. This also allows Cypress to automatically parallelize the cypress tests into four separate machines. 
```

Comment:
Might want to say "HTML screenshot" since you can inspect/click things on the screenshot.

Reply:
Can you inspect the screenshot on cypress cloud ?

Reply:
Yeah, that's how I got error messages for fixing the accessibility test.

Reply:
how, is it not a png?

Reply:
Never mind, it is only not a png on local.